### ISSUE TITLE
Different accuracy when --multigpu True

### ISSUE THREAD

**Issue by GloamXun** [mxbastidasr/DAWN_WACV2020#9]
Hello, I have been training the model on an 8 GPU server using the CIFAR-100 dataset. I followed the instructions in the README.md and noticed a significant difference in accuracy based on the --multigpu setting. Specifically, when --multigpu is set to False, the accuracy is around 71%, but when I set --multigpu to True, the accuracy drops to approximately 64%. Could you provide any insight into why this might be happening? Or, could you please guide me on how you train the model on multiple GPUs?

**Comment by beltegeuse** [comment]
Hello,

I apologize for the delay in my response. I didn't notice your message earlier. When training on multiple GPUs, there is usually a problem with approximating the gradient to make the training parallel. Additionally, certain layers, such as batch norm, are managed differently which can have an impact on the accuracy of the training.

A possible solution to this problem is to use DDP. You can find more information about it at https://pytorch.org/docs/stable/notes/ddp.html. In the code, DataParallel is used as well.

Let me know if you have any further questions or concerns. I will close the issue for now.
